res/css/views/rooms/_EventTile.pcss

```diff
@@ -551,6 +551,18 @@ $left-gutter: 64px;
             background-color: $codeblock-background-color;
         }
 
+        code:not(pre *) {
+            background-color: $inlinecode-background-color;
+            border: 1px solid $inlinecode-border-color;
+            border-radius: 4px;
+            // The horizontal padding is added by gfm.css .markdown-body
+            padding: 2px 0;
```

Comment:
Can you use the spacing vars everywhere? Like for instance `$spacing-2` here.

Reply:
Yes, good idea!